lib/new_relic/agent/instrumentation/resque_helper.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+module NewRelic
```

Comment:
I think it'd make sense to move this helper into the `lib/new_relic/agent/instrumentation/resque/` directory, and name it just helper. I know not all the helpers have been moved into directories like that, but it's a pattern we tried to follow starting when we did the prepend work.